progress-reports/icon-bridge/BTP-September-2022-Progress-Update.md

```diff
@@ -0,0 +1,54 @@
+| Script to change javascore deployer | Released | Script to update SC owners| https://github.com/icon-project/icon-bridge/issues/395 |
+| Local build script update| Released | missing dependencies need to be added | https://github.com/icon-project/icon-bridge/issues/426| 
+| Upgrade(BTS SCORE) | Released on Testnet | upgrade script on bsc testnet for BTS Core. Investingating issue on Mainnet deployment | https://github.com/icon-project/icon-bridge/issues/467 | 
+| Security Audit ticket - syncVerifier for ICON should also verify the headers before updating like BSC | Released | The syncVerifier function is used to sync up the headers from the hashes in bmr config file | https://github.com/icon-project/icon-bridge-planning/issues/50 | 
+| Automate configuration on e2e | Released |  Configuration APIs added for e2e tests to run | https://github.com/icon-project/icon-bridge/issues/415 |
+| Set truffle version in local build script | Released | Set truffle version to v5.5.5 | https://github.com/icon-project/icon-bridge/issues/435 |
+| Alternative build process for local deployment | Released | Update local deployment scripts | https://github.com/icon-project/icon-bridge/issues/463 |
+| Add bsc header and receipts verification | Released | Add block header and receipts verification logic to bsc verifier | https://github.com/icon-project/icon-bridge/issues/377 |
+| Add script to generate e2e config during deployment | Released | Update script so necessary e2e config should be auto-generated by the deployment script itself | https://github.com/icon-project/icon-bridge/issues/494 |
+| Pipeline for notifying of most recent stable contract addresses for testnet and mainnet | Released | List and update the most stable contract addresses | https://github.com/icon-project/icon-bridge/issues/37 |
+| Implement BSC block verification | Released | Relay to verify correctness of incoming messages the same way that the BMV does | https://github.com/icon-project/icon-bridge/issues/35 |
+| Add unit tests to bsc verifier | Released | Add unit tests to bsc verifier | https://github.com/icon-project/icon-bridge/issues/499 |
+| Use god wallet path on e2e config by default | Released | Simplify the process of running e2etests after a successful build process | https://github.com/icon-project/icon-bridge/issues/525 |
+| Add setter getter tests for javascore BMC | Released | Add unit test for all setter and getter methods in javascore BMC | https://github.com/icon-project/icon-bridge/issues/542 |
+| Documentation Updates | Released 1st Draft | Icon-Bridge repo documentation udpate | https://github.com/icon-project/icon-bridge-planning/issues/24 |
+| Support NEAR integration | In Progress | Support NEAR integration team with integration and support | - |
+| Support SNOW integration | In Progress | Support SNOW integration team with integration and support | - |
+| Security Audit | In Progress | Support Security Auditors | - |
+
+
+
+## Deliverables Projected for Next Cycle
```

Comment:
Please add about updating scripts for integrating Harmony, Ethereum, etc.

I assume xcall spec will not make it to this cycle.

Reply:
Xcall planning and estimation should be happening this cycle, so we should include that activity in the report.

Reply:
Ok, in that case just add it as "Planning" in development state